services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+    return movie
+
+
+def get_movies_sessions(session_date: datetime.date = None) -> QuerySet:
+    session = MovieSession.objects.all()

```

Comment:
It would be better to use plural form
```suggestion
    sessions = MovieSession.objects.all()

```